### PR TITLE
feat(classes): use pirate style examples

### PR DESCRIPTION
**Pull request by aemkei**

### CODE REVIEW

**Comment by aemkei** [comment]
Split up the examples and make sure they work.

File: README.md [hunk 1]
```diff
@@ -72,23 +72,54 @@ nums.forEach(v => {
...
+}
+
+var bill = new Pirate('Bill'),
+  jack = new Captain('Jack'),
+  rounds = 20;
+
+while (rounds) {
+  bill.drink();
+  jack.drink();
+}
```
Comment (ro-ka) [inline README.md:128]: Missing something like `rounds--;`.

File: README.md [hunk 2]
```diff
@@ -72,23 +72,54 @@ nums.forEach(v => {
...
+
+bill.isDrunk(); // returns true
+jack.isDrunk(); // returns false
+
+```
+
+Also, there are getters and setters:
+
+```js
+class Pirate {
```
Comment (ro-ka) [inline README.md:142]: For copy-pasting, this example misses a construcor and an `init` of Jack.

**Comment by ro-ka** [comment]
:checkered_flag:

File: README.md (hunk 2 above)
Comment (aemkei) [inline README.md:142]: Don't think, a constructor is required. But yes, we should add one.

**Pull request merged**